Examples_Nuget/ArrayShareExample/ArrayShareExample.csproj

```diff
@@ -9,6 +9,9 @@
   <ItemGroup>
     <PackageReference Include="TileDB.Cloud" Version="0.2.4" />
     <PackageReference Include="TileDB.CSharp" Version="5.2.4" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
+    <PackageReference Include="RestSharp" Version="105.2.3" />
+    <PackageReference Include="Polly" Version="7.2.3" />
```

Comment:
@teo-tsirpanis was this an intentional result of #22? Just curious, I couldn't get this or `QuickstartUdf` to build without adding these references.